stable/README.md

```diff
@@ -1,17 +1,20 @@
-# Getting Started with NuoDB/Helm on OpenShift
+### The instructions on this page are in two parts:
```

Comment:
this page is specifically for OCP. It does not apply at all to vanilla k8s. You removed that important bit of information

Reply:
The whole idea is to update the pages for all our supported K8s. The part of the page that is RH OCP specific has been noted.